rwatch/ui/layer/status_bar_layer.c

```diff
@@ -32,7 +32,7 @@ StatusBarLayer *status_bar_layer_create(void)
     status_bar->layer = layer;
     status_bar->background_color = GColorDarkGray;
     status_bar->text_color = GColorWhite;
-    status_bar->separator_mode = StatusBarLayerSeparatorModeDotted;
+    status_bar->separator_mode = (StatusBarLayerSeparatorMode*)StatusBarLayerSeparatorModeDotted;
```

Comment:
Another case of a different problem. `separator_mode` shouldn't be a pointer in the first place.

Reply:
Agreed